src/main/java/logisticspipes/proxy/buildcraft/BCRenderTESR.java

```diff
@@ -10,32 +12,37 @@
 
 public class BCRenderTESR implements IBCRenderTESR {
 
-    private final Method renderGatesWires;
-    private final Method renderPluggables;
+    private final MethodHandle renderGatesWires;
```

Comment:
Make these handles `static final`, and initialize them in a `static {...}` intialization block - this will improve invokeExact performance even further

Reply:
Done. Thanks for the review and the offline JIT explanation!